File: CódigoAula_04_06.08/ProjetoMedicacaoEmC#/Model/Utilidades.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ProjetoMedicacaoEmC_.Model;

internal class Utilidades
{
    public static void PopularLista(List<int> lista,long quantidade, int inicio,int fim,Boolean aleatorio)
    {
        Random gerador = new Random();
        if (aleatorio)
        {
            for(long i = 0;i < quantidade; i++)
            {
                lista.Add(gerador.Next(inicio, fim + 1));
            }
        }
        if (!aleatorio)
        {
            for(long i = 0; i< quantidade; i++)
            {
                lista.Add(inicio + (int)i);
            }
        }
    }
}

File: CódigoAula_04_06.08/ProjetoMedicacao/Model/Utilidades.java
package Model;
import java.util.List;
import java.util.Random;
public class Utilidades {
    /**
     * método de classe que popula lista ou com números aleatorios e de forma sequencial dentro de uma faixa
     * @param lista
     * @param quantidadeNumeros
     * @param inicio
     * @param fim
     * @param aleatorio - se true os números devem ser aleatórios
     */
    public static void popularLista(List<Integer> lista, long quantidadeNumeros, int inicio, int fim, boolean aleatorio) {
        Random gerador = new Random();
        
        if (aleatorio) {
            for (long i = 0; i < quantidadeNumeros; i++) {
                lista.add( gerador.nextInt(inicio, fim) );
            }
        }

        if (!aleatorio) {
            for (long i = inicio; i < quantidadeNumeros; i++) {
                lista.add( (int)i );
            }
        }
    }

    
}
